phpunit.xml.dist

```diff
@@ -1,17 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit bootstrap="vendor/silverstripe/framework/tests/bootstrap.php" colors="true">
-
-	<testsuite name="Default">
-		<directory>tests</directory>
-	</testsuite>
-
-	<filter>
-		<whitelist addUncoveredFilesFromWhitelist="true">
-			<directory suffix=".php">.</directory>
-			<exclude>
-				<directory suffix=".php">tests/</directory>
-			</exclude>
-		</whitelist>
-	</filter>
-
+    <testsuites>
+        <testsuite name="postrgresql">
+            <directory>tests</directory>
+        </testsuite>
+        <testsuite name="framework">
+            <directory>vendor/silverstripe/framework/tests/php</directory>
```

Comment:
The reason these aren't working is because they're being deleted https://github.com/silverstripe/gha-ci/blob/1/.github/workflows/ci.yml#L488

Simple solution here is to just update gha-ci and add in `|postgresql|sqllite3` in there.  Even though that block is weird and irrelevant for CMS 5, we should still keep it for CMS 4, there's probably still some edge cases out there that require this, particularly when the community uses this CI.